main/src/cgeo/geocaching/models/Geocache.java

```diff
@@ -1869,16 +1869,15 @@ public boolean hasStaticMap() {
     }
 
     /**
-     * Add spoilers stored locally in <tt>/sdcard/GeocachePhotos</tt>. If a cache is named GC123ABC, the
-     * directory will be <tt>/sdcard/GeocachePhotos/C/B/GC123ABC/</tt>.
+     * Add spoilers stored locally in <tt>/sdcard/cgeo/GeocachePhotos</tt>. If a cache is named GC123ABC, the
+     * directory will be <tt>/sdcard/cgeo/GeocachePhotos/C/B/GC123ABC/</tt>.
```

Comment:
As far as I could find it, it's from a GSAK macro. I also found the corresponding issues and commits here: https://github.com/cgeo/cgeo/issues/4957#issuecomment-284263660

When I look into the GSAK macro it is actually `Garmin\GeocachePhotos`. So the user of this macro need to copy the subfolder anyhow. I don't know if this directory might be used by some other Apps in parallel. Should we keep it one level up?